nemo/collections/asr/data/data_simulation.py

```diff
@@ -0,0 +1,1324 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import warnings
+
+import numpy as np
+import pyroomacoustics as pra
```

Comment:
For pyroomacoustics and gpuRIR, these are not included in requirements.txt.

So please do this as [here](https://github.com/NVIDIA/NeMo/blob/987674e29ea90f9a2f663bf95d74bd947d76bbc0/nemo/collections/asr/parts/utils/diarization_utils.py#L36) :
```
try:
    import arpa

    ARPA = True
except ImportError:
    ARPA = False

try:
    import diff_match_patch

    DIFF_MATCH_PATCH = True
except ImportError:
    DIFF_MATCH_PATCH = False

```

And add error handling lines in the code in the methods (functions).
For example, 
 ```
if not PYROOMACOUSTICS:
    raise ImportError("pyroomacoustics should be installed to run this function ~ ")
```
This is because we want all the python modules imported without error message on dependency.